docker/Dockerfile

```diff
@@ -19,5 +19,8 @@ RUN cd /build && \
 	cd .. && \
 	rm -rf /build
 
+COPY pg_doom/pg_doom_client.sh .
```

Comment:
There is no need to copy scripts, because they copied (see Makefile) into binary directory of PostgreSQL during `make install`.